pkg/util/helper/binding_test.go

```diff
@@ -0,0 +1,75 @@
+package helper
+
+import (
+	"testing"
+
+	workv1alpha2 "github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
+)
+
+func TestHasScheduledReplica(t *testing.T) {
+	tests := []struct {
+		name           string
+		scheduleResult []workv1alpha2.TargetCluster
+		want           bool
+	}{
+		{
+			name: "assigned replicas for a cluster",
```

Comment:
The name of sub-test should keep unique for distinguishing them from each other.

Reply:
Fixed. @RainbowMango